feat: File Input (#1218)
Amsterdam/design-system: packages/css/src/components/file-input/README.md

<!-- @license CC0-1.0 -->

# File Input

> The file input component is a styled input element that allows users to select a file from their device.

Reviewer: ```suggestion
Allows the user to upload one or more files from their device.
```
